clients/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java

```diff
@@ -28,13 +28,17 @@
   /**
    * Config property name ({@value #CONF_NESSIE_USERNAME}) for the user name used for (basic)
    * authentication.
+   *
+   * @deprecated "basic" HTTP authentication is not considered secure.
```

Comment:
nit: maybe add `Please use other authentication mechanisms, such as XYZ`

Reply:
Good point. Redirected to `BEARER` in f85e83986ea536bb39fe2c16eee11c6e02d1cc8f